riotfile.py

```diff
@@ -12,7 +12,7 @@
             },
             pys=[3.6, 3.7, 3.8, 3.9],
             pkgs={
-                "pytest": latest,
+                "pytest": "==6.1.2",
```

Comment:
tbh not a fan of pinning everything.
Who's gonna upgrade those?

Reply:
I think I pinned it because some of the tests assert on the pytest output format which could change in newer versions of pytest

Reply:
I'm ok to not pin it - just might make the tests brittle. The better approach might not be dependent on `pytest` in the tests...